examples/python/multiplex/route_guide_pb2_grpc.py

```diff
@@ -0,0 +1,121 @@
+import grpc
+from grpc.framework.common import cardinality
+from grpc.framework.interfaces.face import utilities as face_utilities
+
+import route_guide_pb2 as route__guide__pb2
```

Comment:
@soltanmm-google: this is [issue 9058](https://github.com/grpc/grpc/issues/9058), right? Do we know when it might happen?

@kpayson64: I'd like to move forward with this now given that I think there's a much greater difference between what's in source control now and this draft of the code than there is between this draft of the code and what users will see when the import duplication problem is fixed. Also yes to another example code update after the import duplication problem is fixed.